pkg/api/mlflow/service/artifact/validators.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // RelativePathRegExp checks for the `../` in provided path.
-var RelativePathRegExp = regexp.MustCompile(`\.{2}\/`)
+var RelativePathRegExp = regexp.MustCompile(`\.{2}\/|^\.{2}$`)
```

Comment:
`foo..` will be considered invalid — I think you may want to just do a split and check for directories that are named exactly `..` instead of the regex

Reply:
never mind the above!

Reply:
`./..` would not be blocked by this, I think

Reply:
no no, foo.. it should work properly.

Reply:
yes yes. tests are already include such a case.